### PR TITLE
Repair error in uVariant

### PR DESCRIPTION
**Pull request by rain-on**

### CODE REVIEW

File: core/src/test/java/tech/pegasys/eth2signer/core/signing/FilecoinAddressTest.java [hunk 1]
```diff
@@ -89,6 +90,12 @@ void addressWithInvalidLengthThrowsError() {
         .hasMessage("Invalid Address length");
   }
 
+  @Test
+  void thinkThiIsBad() {
```
Comment (usmansaleem) [inline core/src/test/java/tech/pegasys/eth2signer/core/signing/FilecoinAddressTest.java:94]: may be rename method to "decodeShouldNotFail" ?

**Review by usmansaleem — APPROVED** [review]
LGTM

File: core/src/test/java/tech/pegasys/eth2signer/core/signing/FilecoinAddressTest.java (hunk 1 above)
Comment (rain-on) [reply core/src/test/java/tech/pegasys/eth2signer/core/signing/FilecoinAddressTest.java]: heh - I've removed it as it was too ahrd to justify as a reference test

File: acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/comparison/CompareSignMessageAcceptanceTest.java [hunk 2]
```diff
@@ -107,8 +111,20 @@ private void testRandomMessage() {
...
+                      "Signature Comparison failed from msg = "
+                          + jsonRequest
+                          + ". With addr = "
+                          + address)
                   .isEqualToComparingFieldByField(signerFcSig.getSignature());
             });
   }
+
+  @Test
+  public void sendRawJsonMessage() throws JsonProcessingException {
```
Comment (jframe) [inline acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/comparison/CompareSignMessageAcceptanceTest.java:124]: this test seems redundant with compareRandomMessageSignaturesBetweenLotusAndEthSigner is it needed?

File: acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/comparison/CompareSignMessageAcceptanceTest.java [hunk 3]
```diff
@@ -41,33 +40,34 @@ void initSigner() {
 
   @Test
   void compareRandomMessageSignaturesBetweenLotusAndEthSigner() {
+    final int signatureCount =
```
Comment (jframe) [inline acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/comparison/CompareSignMessageAcceptanceTest.java:43]: why does this need to be a sys property, is this going to be overridden in circleCI?

File: acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/comparison/CompareSignMessageAcceptanceTest.java (hunk 2 above)
Comment (rain-on) [reply acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/comparison/CompareSignMessageAcceptanceTest.java:124]: It is redundant. BUT - in the future, if a SignMessage fails - we can easily cut-and-paste the transmission body here and re-run it - its more of a template (can be removed ... might be overkill)

File: acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/comparison/CompareSignMessageAcceptanceTest.java (hunk 3 above)
Comment (rain-on) [reply acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/comparison/CompareSignMessageAcceptanceTest.java:43]: 🤷 Having it as an arbitrary constant seemed odd - this allows a degree of flexibility - happy to revert ...

**Pull request merged**